WalletWasabi.Fluent/ViewModels/Wallets/Home/History/HistoryViewModel.cs

```diff
@@ -10,56 +10,31 @@
 using ReactiveUI;
 using WalletWasabi.Blockchain.Analysis.Clustering;
 using WalletWasabi.Fluent.Helpers;
-using WalletWasabi.Fluent.Models.UI;
 using WalletWasabi.Fluent.Models.Wallets;
 using WalletWasabi.Fluent.TreeDataGrid;
 using WalletWasabi.Fluent.ViewModels.Wallets.Home.History.HistoryItems;
 using WalletWasabi.Fluent.Views.Wallets.Home.History.Columns;
+#pragma warning disable CA2000
```

Comment:
I am not sure if this is ok to disable it in the whole file, btw removing it Rider doesn't show any new warning to me.